tools/seccomp_util/src/main.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2019 - 2020 ESRLabs
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+use anyhow::Result;
+use itertools::Itertools;
+use regex::Regex;
+use std::{fmt::Write, fs::File, io, io::BufRead, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// Path to strace log file
+    input: PathBuf,
+}
+
+fn main() -> Result<()> {
+    env_logger::init();
+
+    let opt = Opt::from_args();
+    generate(opt.input)
+}
+
+fn generate(strace_file: PathBuf) -> Result<()> {
+    let lines = open_file(strace_file);
+    let names = parse_strace(lines);
+    let entry = gen_seccomp_entry(names).expect("Failed to generate seccomp entry");
+    println!("{}", &entry);
+    Ok(())
+}
+
+fn open_file(strace_file: PathBuf) -> Vec<String> {
```

Comment:
Can we inline those two and be consistent with either expecting or returning an `Result`?